src/index.ts

```diff
@@ -103,21 +103,17 @@ client.on('messageCreate', async (message) => {
     });
     let msgIsCommand = false;
     let prefixLen = 0;
-    const prefix = client.guildData.get(message.guild!.id, 'prefix');
+    const prefix = client.guildData.get(message.guild!.id, 'prefix')!;
+    const lowercasedMessageContent = message.content.toLowerCase();
 
-    if (message.content.toLowerCase().startsWith(prefix)) {
-        msgIsCommand = true;
+    if (lowercasedMessageContent.startsWith(prefix)) {
         prefixLen = prefix.length;
```

Comment:
```suggestion
        msgIsCommand = true;
        prefixLen = prefix.length;
```
Forgot to set this...

Reply:
nvm ig